src/js/hyperschedule.js

```diff
@@ -440,10 +454,37 @@ function generateCourseDescription(course)
 
 function getCourseColor(course, format = "hex")
 {
+  let hue = "random";
+  let seed = CryptoJS.MD5(course.courseCode).toString();
+
+  if (course.starred || !courseInSchedule(course))
+    switch (gGreyConflictCourses) {
+      case gGreyConflictCoursesOptions[0]:
+        break;
+
+      case gGreyConflictCoursesOptions[1]:
+        if (course.starred && courseConflictWithSchedule(course)) {
+          hue = "monochrome";
+          seed = "-10";
+        }
+        break;
+
+      case gGreyConflictCoursesOptions[2]:
+        if (courseConflictWithSchedule(course)) {
+          hue = "monochrome";
+          seed = "-10";
```

Comment:
Where did `-10` come from? Shouldn't we just hardcode a specific color rather than randomly generating always the same color?

Reply:
I just did this so that different color formats could be returned and I'm unsure how else to implement this?

Reply:
Oh, I see. I didn't realize `format` could be different. Fair enough :)